### PR TITLE
FARIS BUGFIX EXTRAVAGANZA 2

### LINKED ISSUE #185: Level Progress Section on Homepage Should Do Something
either link to your profile or leaderboard, whichever makes more sense

### LINKED ISSUE #200: Inconsistency in Admin Settings
Very low priority but I only have store admin, the button for event admin shows for me even tho user points/attendance shows as restricted: 
Screenshot 2024-03-29 at 11 42 06 PM

### PR DESCRIPTION
**Pull request by alexzhang1618**
stuff in #202 not related to query params that still need to be fixed

Closes #185
Closes #200

### CODE REVIEW

File: src/components/events/EventCard/style.module.scss
Comment (SheepTester) [inline src/components/events/EventCard/style.module.scss:1]: https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/74ad042b-76c4-4572-8804-b926bda522e2

the outline width does not seem to be transitioning out. and on transition in, it kind of looks like the outline color is starting from white

maybe you could try doing `outline: 0 solid ...;` as a non-hover style, and then on hover, just change `outline-width: 4px;`

**Review by SheepTester — APPROVED** [review]
looks good

**Comment by farisashai** [comment]
Closes #185
Closes #200

**Comment by SheepTester** [comment]
The new transition looks good :shipit:

**Pull request merged**